.github/PULL_REQUEST_TEMPLATE/preproc.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Core contribution
+about: 
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Before you start, read [contributing.md](https://github.com/ESMValGroup/ESMValTool/blob/version2_development/CONTRIBUTING.md).
+If you are making a dataset fix, first read the [documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/esmvalcore/fixing_data.html).
+
+**Tasks**
+-   [ ] Create an issue to explain what you are going to do (and add the link at the bottom)
+-   [ ] Add unit tests
+-   [ ] Public functions should have a numpy-style docstring so they appear properly in the [API documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/api/esmvalcore.html). For all other functions a one line docstring is sufficient.
+-   [ ] If writing a new preprocessor function, please update the [documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/esmvalcore/preprocessor.html)
+-   [ ] Make sure the Circle/CI tests pass
+-   [ ] Make sure the Codacy tests pass
+-   [ ] Please use `yamllint` to check that your YAML files do not contain mistakes
+-   [ ] If you make backward incompatible changes to the recipe format, make a new pull request in the [ESMValTool repository](https://github.com/ESMValGroup/ESMValTool) and add the link below
```

Comment:
Maybe also "Assign the author(s) of the affected recipe(s) as reviewer(s)" or similar?

Reply:
Though I guess it makes more sense to add this text in https://github.com/ESMValGroup/ESMValTool/pull/1315

Reply:
I added this to mod_recipe.md in ESMValGroup/ESMValTool#1315. I think that should be enough.